File: Summary/week13_1/app/src/main/java/com/pnu/fashion/MainActivity.java
package com.pnu.fashion;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.util.Pair;

import android.app.Activity;
import android.content.Intent;
import android.graphics.Bitmap;
import android.graphics.ImageDecoder;
import android.net.Uri;
import android.os.Build;
import android.os.Bundle;
import android.os.Environment;
import android.provider.MediaStore;
import android.util.Log;
import android.widget.Button;
import android.widget.ImageView;
import android.widget.TextView;

import com.pnu.fashion.classify.Classifier;

import java.io.IOException;
import java.util.Locale;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        //Log.d(TAG, Environment.getExternalStorageDirectory().getAbsolutePath());
        
        // findViewById() 함수를 통한 DrawView 버튼 연결
        Button drawBtn = findViewById(R.id.drawBtn);
        // 버튼 리스너 설정
        drawBtn.setOnClickListener(view -> {
            // DrawActivity를 실행하는 인텐트 생성
            Intent i = new Intent(MainActivity.this, DrawActivity.class);
            // StartActivity() 함수에 인텐트를 전달
            startActivity(i);
        });
    }
}

File: Summary/7_1Fragment/MainActivity.kt
package com.example.android7week

import androidx.appcompat.app.AppCompatActivity
import android.widget.Button

import android.os.Bundle
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentTransaction

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val fragButton = findViewById<Button>(R.id.fragBut1)
        // Fragment 제어 객체 선언
        val fragmentManager: FragmentManager = supportFragmentManager
        var onClicked = false
        fragButton.setOnClickListener{
            if (onClicked) {
                onClicked = false
	    // Fragment Transaction 객체 선언
                val transaction: FragmentTransaction = fragmentManager.beginTransaction()
	    // Fragment 제거
                val frameLayout = supportFragmentManager.findFragmentById(R.id.fragment_content)
                transaction.remove(frameLayout!!).commit()
            }
            else {
                onClicked=true
                val transaction: FragmentTransaction = fragmentManager.beginTransaction()
	    // Fragment 추가
                transaction.add(R.id.fragment_content, OneFragment()).commit()
            }
        }
    }
}

